src/plugins/intel_gpu/tests/test_cases/non_max_suppression_test.cpp

```diff
@@ -4,17 +4,23 @@
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
-#include "test_utils.h"
-
 #include <intel_gpu/primitives/data.hpp>
 #include <intel_gpu/primitives/input_layout.hpp>
 #include <intel_gpu/primitives/mutable_data.hpp>
 #include <intel_gpu/primitives/non_max_suppression.hpp>
 
+#include "test_utils.h"
+
 using namespace cldnn;
 using namespace ::tests;
 
-template <typename T>
+template <typename T, cldnn::format::type l>
+struct TypeWithLayout {
```

Comment:
Maybe better name would be TypeWithFormat? Notion layout contains format and other attributes.